docs/linux/sql-server-linux-setup-sql-agent.md

```diff
@@ -19,6 +19,9 @@ ms.assetid: 77f16adc-e6cb-4a57-82f3-7b9780369868
 - [For Versions 2017 CU4 and above, Enable the SQL Server Agent](#EnableAgentAfterCU4)
 - [For Versions 2017 CU3 and Below, Install the SQL Server Agent](#InstallAgentBelowCU4)
 
+> [!NOTE]
```

Comment:
I think it would be more consistent to put this note below the command to enable SQL Agent, after current line 33.

Also wouldn't `/etc/hosts` need to be wrapped in the code tags?

Reply:
I think it would also be good to add context to the reasoning behind checking that file. Since it is based around the Agent not starting, reference it as such.